unity-renderer-desktop/Assets/Scripts/MainScripts/DCL/MainDesktop/MainDesktop.cs

```diff
@@ -46,9 +46,16 @@ protected override void InitializeCommunication()
             //              IntegrationTestSuite_Legacy base class.
             if (!Configuration.EnvironmentSettings.RUNNING_TESTS)
             {
+                var withSSL = true;
                 int startPort = CommandLineParserUtils.startPort;
+                
+                #if UNITY_EDITOR
+                withSSL = false;
```

Comment:
```suggestion
                withSSL = DebugConfigComponent.i.webSocketSSL;
```

Can you make this change?

Reply:
Yes! totally agree